index.html

```diff
@@ -1876,6 +1876,76 @@ <h3>Examples</h3>
       </pre>
     </section>
   </section>
+  <section>
+    <h2>Camera Effect Status</h2>
+    <p>The {{VideoFrameMetadata}} interface exposes the effect state as a property,
+      which allows apps to know the state for every frame. This is important for
+      scenarios where the app must ensure user privacy by never sending an
+      un-blurred frame off of the user's device.</p>
+    <section>
+      <h3>{{VideoFrameMetadata}}</h3>
+      <pre class="idl">
+partial dictionary VideoFrameMetadata {
+  BackgroundBlur background_blur;
+};</pre>
+      <section class="notoc">
+        <h4>Members</h4>
+        <dl class="dictionary-members" data-link-for="VideoFrameMetadata" data-dfn-for="VideoFrameMetadata">
+          <dt><dfn><code>background_blur</code></dfn> of type <code>{{BackgroundBlur}}</code></dt>
+          <dd>
+            <p>The state of the background blur effect for the current frame.</p>
+          </dd>
+        </dl>
+      </section>
+    </section>
+    <section>
+      <h3>{{BackgroundBlur}}</h3>
+      <pre class="idl">
+dictionary BackgroundBlur {
+  boolean enabled;
```

Comment:
I think this should be a required property.  If `background_blur` is present, `enabled` should have a value.